### DISCUSSION TITLE
How to use whisper.cpp in Vue3+Vite?

### DISCUSSION THREAD

**Discussion by YolineWu** [ggml-org/whisper.cpp#1378] [Q&A]
I have limited knowledge about ES6 or about vite and I'm unsure how to use the 'whisper.cpp' library in the context of Vue 3 + Vite. Can anyone provide guidance on whether this library is compatible with Vue 3 + Vite, and if so, how to integrate it?

**Supplementary Information:**

I have attempted to use [the 'whisper.cpp' npm library](https://www.npmjs.com/package/whisper.cpp). Initially, I encountered the error message: `GET http://localhost:5173/libwhisper.worker.js 404 (Not Found)`. To address this, I employed the [vite-plugin-static-copy](https://www.npmjs.com/package/vite-plugin-static-copy) library to copy 'libwhisper.worker.js' from the 'whisper.cpp' package directory to the 'public' directory, which resolved the issue.

However, I then encountered another error: `worker sent an error! http://localhost:5173/libwhisper.worker.js:1: Uncaught TypeError: Failed to execute 'createObjectURL' on 'URL': Overload resolution failed.`

Could anyone provide guidance on how to resolve this error? Or, is there a different approach I should be using?

**Comment by niclan** [comment]
You should probably contact the people that wrote the whisper.cpp npm library for ansewers to these questions.
